docs/udp.rst

```diff
@@ -433,3 +433,131 @@ which we can process further and download::
     cube.download("fancy.tiff", format="GTiff")
 
 See :ref:`datacube_from_process` for more information on :func:`~openeo.rest.connection.Connection.datacube_from_process`.
+
+
+.. _workflow_example:
+
+Working Example: Create EVI Timeseries UDP
+==========================================
+
+In this first example, we'll create an openEO workflow script,
+including declaring parameters, building, storing and loading a UDP
+
+Here, let us consider that we want to recalculate the EVI-timeseries
+as done in :ref:`example-use-case-evi-map-and-timeseries` but for a different area.
+Therefore for future, instead of redefining the whole process, we simply 
+want to define it as a UDP ``EVI_timeseries``, that can be easily shared 
+and reused later without dealing with the entire workflow each tim:
+
+.. code-block:: python
+
+    import openeo
+    from openeo.api.process import Parameter
+
+    # Create connection to openEO back-end
+    connection = openeo.connect("...").authenticate_oidc()
+
+    # declaring parameters
+    temporal_interval = Parameter(
+        name="temporal_interval",
```

Comment:
I would name this parameter `temporal_extent` iso `temporal_interval` to stay closer to the standard